### PR TITLE
140/modal-and-fixes

### PR DESCRIPTION
**Pull request by ail3ngrimaldi**
This PR adds a modal so that people can know if their emails had been registered or not.

I was able to test only if it failed, so in order to test the success one, we have to pass the last airtable keys during the deployment process. 

In addition I:

- Deleted repetead list-style: none declarations and set it as default on common.css file.
- Fixed color from about.liquid dropdown menu, and removed some extra spaces in the CSS styles.
- Added a background image for smaller screens on services - #key_feature.

### CODE REVIEW

File: index.liquid [hunk 1]
```diff
@@ -5,6 +5,14 @@ data:
   route: home
 ---
 
+<div id="modal">
```
Comment (olanod) [inline index.liquid:8]: Back in my days it was too early to use `<dialog>` but nowadays it seems to have [great support](https://caniuse.com/?search=dialog), dialog elements are much better for modals, starting with the fact they come with the `.showModal()` function.

Comment (ail3ngrimaldi) [reply index.liquid]: Done, Im available for any other change needed,  hoping we can launch this landing page today! :)

File: index.liquid [hunk 2]
```diff
@@ -220,21 +217,19 @@ const baseId = '{{ env.AIRTABLE_BASE_ID }}';
...
+	modal.showModal();
 }
 
-function closeModal() {
-    modal.style.display = 'none';
-}
-
-closeButton.addEventListener('click', closeModal);
+closeButton.addEventListener('click', () => {
+	modal.close()
```
Comment (olanod) [inline index.liquid:231]:
```suggestion
    modal.close()
```

File: index.liquid [hunk 3]
```diff
@@ -246,9 +263,11 @@ mailForm.addEventListener('submit', async (event) => {
     }
 
     const result = await response.json();
+	openModal(`<img src="/img/success.svg" alt="success" /><h2>¡Gracias!</h2><h3>Su mail ha sido registrado exitosamente.</h3>`);
```
Comment (olanod) [inline index.liquid:266]:
```suggestion
    openModal(`<img src="/img/success.svg" alt="success" /><h2>¡Gracias!</h2><h3>Su mail ha sido registrado exitosamente.</h3>`);
```

File: index.liquid [hunk 4]
```diff
@@ -246,9 +263,11 @@ mailForm.addEventListener('submit', async (event) => {
...
     console.log('Datos enviados exitosamente a Airtable:', result);
 
   	} catch (error) {
+		openModal('<img src="/img/error.svg" alt="error" /><h2>Error</h2><h3>Hubo un problema al registrar su mail.</h3><h3 class="bolder">Por favor, intente nuevamente.</h3>');
```
Comment (olanod) [inline index.liquid:270]:
```suggestion
        openModal('<img src="/img/error.svg" alt="error" /><h2>Error</h2><h3>Hubo un problema al registrar su mail.</h3><h3 class="bolder">Por favor, intente nuevamente.</h3>');
```

**Comment by olanod** [comment]
Replaced tabs for spaces to be consistent with the rest of the JS code.

**Pull request merged**